src/osg2vsg/GeometryUtils.cpp

```diff
@@ -0,0 +1,90 @@
+#include <osg2vsg/GeometryUtils.h>
+
+namespace osg2vsg
+{
+
+    vsg::ref_ptr<vsg::vec2Array> convertToVsg(osg::Vec2Array* inarray)
+    {
+        vsg::ref_ptr<vsg::vec2Array> outarray(new vsg::vec2Array(inarray->size()));
+        for (auto i = 0; i < inarray->size(); i++)
```

Comment:
auto i = 0 will likely map the i to an int, but size() will return a std::size_type so my guess is we'll get warnings on some platforms.  I'll fix this after merging.